api/src/main/resources/liquibase-update-to-2.0.xml

```diff
@@ -9485,6 +9491,8 @@
     </changeSet>
 
     <changeSet id="TRUNK-3422-20160216-1704" author="PralayRamteke" >
+		<validCheckSum>3:b4caee668484f1d2759bbf3feeba2daa</validCheckSum> <!-- old checksum-->
+		<validCheckSum>3:13b334dbd8fba2fea886d1c9aab40e16</validCheckSum> <!-- current checksum 20181107-4-->
```

Comment:
Are you sure the current checksum for this is? 20181107-4

Reply:
Wait, do I need to comment it like `<!-- current checksum 3:13b334dbd8fba2fea886d1c9aab40e16 -->?`

Reply:
If it is a repetition of what is already there, then it does not add value.

Reply:
If you want to teach yourself regarding how to comment on these, just search in the same file for all instances of "validCheckSum" to see how they look like.